webhooks-extension/src/components/WebhookDisplayTable/WebhookDisplayTable.js

```diff
@@ -191,28 +342,36 @@ export class WebhookDisplayTable extends Component {
               )}
               />
           </div>
+
+          <div className="modal">
+            <Modal open={this.state.showDeleteDialog}
+              className=".delete-modal"
+              modalLabel="Are you sure?"
```

Comment:
I would hide the label if possible ... I don't think it is adding anything

Reply:
I agree, it's now gone